test/cypress/integration/a11y.cy.js

```diff
@@ -75,7 +75,7 @@ describe('a11y', () => {
 
       cy.document().then(() => {
         cy.wait(1000);
-        cy.get('.shepherd-element').tab().tab().tab().tab().tab().tab();
+        cy.get('.shepherd-content').tab().tab();
```

Comment:
Does this not work with all the previous tabs? This test is meant to tab a ton of times to show it is focus trapped in the modal

Reply:
It did not work, I can test some more though. It just didn't work as is, but now that I understand the point more I'll try again. Cannot be from the DIALOG though, so the content selector is needed.

Reply:
We definitely want to make sure we keep the spirit of this test. It should tab at least like 4-6 times I think as it should select like the title, description, next button, previous button, etc and we want to make sure if you tab more times than the things inside that it loops around and stays focus trapped.

Reply:
We can pair and I can definitely show you that it does, so the correct way would be to make sure it loops around enough times to stay on the element with the data test attribute?

Reply:
It just needs to prove that it loops around and does not start hitting elements outside of the tour

Reply:
finally got this working by adding some more to make sure it loops all the focus locked elements.